api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/data/quality/Ec15PatientListDataset.java

```diff
@@ -33,6 +33,11 @@ public Ec15PatientListDataset(HivMetadata hivMetadata) {
     this.hivMetadata = hivMetadata;
   }
 
+  /**
+   *
+   * @param parameterList
```

Comment:
Did we need to make changes to this file?

Reply:
@KelvinWanyama any comment here?

Reply:
No @ningosi and @nthfloor . I just added the comments